cms/models.py

```diff
@@ -17,6 +17,15 @@
 from profiles.api import get_social_username
 from ui.views import get_bundle_url
 
+def program_info(programs):
```

Comment:
Can we use the existing `ProgramSerializer` for this? It provides the same two fields. You should be able to use it like `ProgramSerializer().to_representation(program)`